src/server/generic_family.cc

```diff
@@ -644,13 +644,14 @@ void OpScan(const OpArgs& op_args, const ScanOpts& scan_opts, uint64_t* cursor,
   PrimeTable::Cursor cur = *cursor;
   auto [prime_table, expire_table] = db_slice.GetTables(op_args.db_cntx.db_index);
 
-  size_t buckets_iterated = 0;
-  // 10k Traverses
-  const size_t limit = 10000;
+  const auto start = absl::Now();
+  // Don't allow it to monopolize cpu time.
+  const absl::Duration timeout = absl::Milliseconds(200);
```

Comment:
I think 200 mili is a lot here

ScanGeneric have 100 mili break so in this level I think we can do 10 mili

Reply:
sounds good :dancers: